cmd/describe/cluster/cmd.go

```diff
@@ -413,12 +414,103 @@ func run(cmd *cobra.Command, argv []string) {
 			" - Details:                 %s\n",
 			str, reason.Summary(), reason.Details())
 	}
+
+	inflightChecks, err := r.OCMClient.GetInflightChecks(cluster.ID())
+	if err != nil {
+		r.Reporter.Errorf("Failed to get inflight checks for cluster '%s': %v", cluster.ID(), err)
+		os.Exit(1)
+	}
+	if len(inflightChecks) > 0 {
+		summaries := []string{}
+		for _, inflight := range inflightChecks {
+			if inflight.State() == "failed" {
+				summary := fmt.Sprintf("\t"+
+					"ID:                 %s\n"+
+					"\tLast run:           %s\n",
+					inflight.ID(), inflight.EndedAt().Format("Jan _2 2006 15:04:05 MST"))
+				details, err := parseInflightCheckDetails(inflight)
+				if err != nil {
+					r.Logger.Errorf("Unexpected error parsing inflight details '%s: %v", inflight.ID(), err)
+					continue
+				}
+				summary += details
+				summaries = append(summaries, summary)
+			}
+		}
+		if len(summaries) > 0 {
+			str += fmt.Sprintf("Failed Inflight Checks:\n%s\n", strings.Join(summaries, "\n"))
+			str += fmt.Sprintf("\tPlease run `rosa verify network -c %s` after adjusting"+
```

Comment:
After tomorrow's discussion, 
should we output this for inflight.Type() == "egress" only?

Reply:
So I think we should be able to resolve all eventually. Like if a user adjusts the subnet tags and has the warning they need to be able to clear up the inflight too

Reply:
Showing only the egress errors now